content/docs/0.8.x/reference/cli/_index.md

```diff
@@ -168,27 +168,33 @@ keptn.exe auth --endpoint=%KEPTN_ENDPOINT% --api-token=%KEPTN_API_TOKEN%
 </p>
 </details>
 
-## Use Keptn CLI with Multiple Contexts
+## Use Keptn CLI with multiple contexts
 
-After authenticating Keptn CLI with multiple Kubernetes clusters, we can directly run Keptn CLI commands in the current context. As soon as we switch the Kube context (e.g. `kubectl config use-context my-cluster-name`), the Keptn CLI will detect the change in the context and ask you to run the command in the changed context or not. If the Keptn CLI is already authenticated for that particular context, the command will run successfully; else it will end up throwing an error, i.e. `Error: credentials not found in native keychain`
+After authenticating the Keptn CLI with multiple Kubernetes clusters, we can directly run Keptn CLI commands in the current context. 
 
-In case of multi-installation of Keptn in the same cluster, we need to pass a flag `--namespace` or `-n` to tell the Keptn CLI to use the credentials for that particular Keptn installation, else it will take the default namespace which is: `keptn`
+* As soon as we switch the Kube context (e.g., by executing: `kubectl config use-context my-cluster-name`), the Keptn CLI will detect the change of the Kube context and ask you to run the command in the changed context or not. 
+
+* If the Keptn CLI is authenticated for that particular context, the command will run successfully; else it will end up throwing the error: `Error: credentials not found in native keychain`
```

Comment:
The error message mentions the `native keychain` but for me it is not clear what that means. later in this document we refer to a `credential manager`. Are these two the same things?

Reply:
I double-checked the code, and the error message is actually another one. It throws the error: `This command requires to be authenticated. See "keptn auth" for details`

Reply:
Hence, the term keychain is not used anymore.